src/templates/homepage/AnnouncementsSection.tsx

```diff
@@ -16,15 +16,20 @@ type TemplateAnnouncementsSectionProps = Omit<
 
 export const TemplateAnnouncementsSection = forwardRef<
   HTMLDivElement,
-  TemplateAnnouncementsSectionProps
+  TemplateAnnouncementsSectionProps & {
+    announcementScrollRefs: Ref<HTMLDivElement>[]
+  }
```

Comment:
nit: prefer to have these on `TemplateAnnouncementsSectionProps`

Reply:
182daa87d8106f4c884cd674b09df274837508aa